.gitattributes

```diff
@@ -1,5 +1,6 @@
 * text=auto
 
+/docs export-ignore
```

Comment:
We don't want to distribute this package with docs?

Reply:
Usually docs are not distributed. They are meant to be read directly here on github